public_dropin_gpu_environments/nemollm_inference_ms/start_server.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+echo "Starting Custom Model environment with NIM"
+set -e
+
+if [ -z "${$MODEL_NAME}" ]; then
+    echo "The MODEL_NAME must be set in runtime parameters"
+fi
+
+
+if [ "${ENABLE_CUSTOM_MODEL_RUNTIME_ENV_DUMP}" = 1 ]; then
+    echo "Environment variables:"
+    env
+
+    echo
+    echo "Running NVIDIA init scripts..."
+    echo
+    /opt/nvidia/nvidia_entrypoint.sh /bin/true
+fi
+
+source /home/nemo/dr/bin/activate
+echo
+echo "Starting DRUM server in background..."
+echo
+nohup drum server --with-triton-server "$@" &
+
+if [[ -e $CODE_DIR/custom.py ]]; then
+    echo
+    echo "Running custom.py..."
+    echo
+    python ${CODE_DIR}/custom.py
+fi
```

Comment:
I don't think we will need this now that we are using DRUM because drum has it's own support for hooks that we can leverage to download models.

Reply:
Now we have two different "custom models" with support of downloading models:
- Triton's custom model which is served by Triton's python backend (e.g. vLLM model, uses python backend)
- DRUM custom model, with hooks.

Reply:
removed this if-block.